org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/vis/StreamView.java

```diff
@@ -45,36 +44,36 @@
 
 /**
  * A view which displays a view of a stream, either MJpeg or Epics array.
- * 
+ *
  * TODO: Matt Taylor this needs to be your absolute best commented and organised code
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)